tests/Integration/Schema/Directives/DeleteDirectiveTest.php

```diff
@@ -245,6 +245,27 @@ public function testDeletesMultipleEmptyInput(): void
         ]);
     }
 
+    public function testDeleteRequiresAtLeastOneArgument(): void
+    {
+        $this->schema .= /** @lang GraphQL */ '
+        type User {
+            id: ID!
+        }
+
+        type Mutation {
+            deleteUser(id: ID @whereKey, email: String @eq): User @delete
+        }
+        ';
+
+        $this->graphQL(/** @lang GraphQL */ '
+        mutation {
+            deleteUser {
```

Comment:
correct me if I'm wrong here but here gets checked if u call the mutation and provide an Id to delete. This was also the case in my example. I passed an Id to that function in #2303 but instead of deleting that one id I passed it deleted everything.

Reply:
Right, this will not specifically solve what went wrong in your case. Perhaps we can also fail during schema validation if a field with `@delete` has arguments without any `ArgBuilderDirective`?

Reply:
Alright, I have implemented a new solution that dynamically checks if there would be any modifications to the query builder.